db/migrations/mysql/000078_create_oauth_mattermostAppID.up.sql

```diff
@@ -0,0 +1,14 @@
+SET @preparedStatement = (SELECT IF(
+    (
+        SELECT COUNT(*) FROM INFORMATION_SCHEMA.COLUMNS
+        WHERE table_name = 'OAuthApps'
+        AND table_schema = DATABASE()
+        AND column_name = 'MattermostAppID'
+    ) > 0,
+    'ALTER TABLE OAuthApps ADD COLUMN MattermostAppID varchar(32);',
+    'SELECT 1'
```

Comment:
We need to execute the false condition here, therefore:
```suggestion
    'SELECT 1',
    'ALTER TABLE Preferences ADD COLUMN MattermostAppID varchar(32);'
```

Reply:
@isacikgoz Thanks for finding that bug :+1: 

Why `ALTER TABLE Preferences`?   Should it not be
```suggestion
    'SELECT 1'
    'ALTER TABLE OAuthApps ADD COLUMN MattermostAppID varchar(32);',
```

Reply:
Another question: What is the idiomatic pattern here? Change the ordering or modify the `if` condition?

Reply:
Ah yes, it's needs to be `OAuthApps`, I was just trying to confirm on my test schema with another table :) For the idiomatic pattern, I don't know the general approach but, to me the condition should check "IF EXIST" rather than "IF NOT EXIST".